src/main/java/se/liu/ida/hefquin/engine/wrappers/graphqlwrapper/impl/RdfViewConfigurationImpl.java

```diff
@@ -0,0 +1,139 @@
+    }
+
+    @Override
+    public boolean containsClassURI(String uri) {
+        if(uri.startsWith(classPrefix)){
+            final String className = uri.substring(classPrefix.length());
+            return classToProperty.containsKey(className);
+        }
+        return false;
+    }
+
+    @Override
+    public boolean containsProperty(String className, String propertyName) {
+        if(containsClass(className)){
+            return classToProperty.get(className).containsKey(propertyName);
+        }
+        return false;
+    }
+
+    @Override
+    public boolean containsPropertyURI(String className, String uri) {
+        if(containsClass(className)){
+            if(uri.startsWith(propertyPrefix)){
+                final String withoutPrefix = uri.substring(propertyPrefix.length());
+                final int suffixIndex = withoutPrefix.lastIndexOf("_of_");
+                if(suffixIndex > 0){
+                    final String propertyName = withoutPrefix.substring(0, suffixIndex);
+                    return containsProperty(className, propertyName);
+                }
+            }
+        }
+        return false;
+    }
+
+    @Override
+    public boolean isObjectProperty(String className, String propertyName) {
+        if(containsClass(className) && containsProperty(className, propertyName)){
+            GraphQLFieldType type = classToProperty.get(className).get(propertyName).getFieldType();
+            return type.equals(GraphQLFieldType.OBJECT);
+        }
+        return false;
+    }
+
+    @Override
+    public final Set<String> getObjectURIs(String className) {
+        Set<String> objectURIs = new HashSet<>();
+        if(containsClass(className)){
+            Map<String,GraphQLViewProperty> properties = classToProperty.get(className);
+            for(String propertyName : properties.keySet()){
+                if(isObjectProperty(className, propertyName)){
+                    objectURIs.add(propertyPrefix + propertyName + "_of_" + className);
+                }
+            }
+        }
+        return objectURIs;
+    }
+
+    @Override
+    public final Set<String> getScalarURIs(String className) {
+        Set<String> scalarURIs = new HashSet<>();
+        if(containsClass(className)){
+            Map<String,GraphQLViewProperty> properties = classToProperty.get(className);
+            for(String propertyName : properties.keySet()){
+                if(!isObjectProperty(className, propertyName)){
+                    scalarURIs.add(propertyPrefix + propertyName + "_of_" + className);
+                }
+            }
+        }
+        return scalarURIs;
+    }
+
+    @Override
+    public final String getPropertyValueType(String className, String propertyName) {
+        if(containsClass(className) && containsProperty(className, propertyName)){
+            return classToProperty.get(className).get(propertyName).getValueType();
+        }
+        return null;
```

Comment:
please state in the javadoc of the interface that this method returns `null` if the property is not found.